src/components/SummaryPanel/ResponseTimeChart.tsx

```diff
@@ -48,14 +53,20 @@ export default class ResponseTimeChart extends React.Component<ResponseTimeChart
             <div>
               <strong>{this.props.label}:</strong>
             </div>
-            <AreaChart
-              size={{ height: 80 }}
-              color={{ pattern: [PfColors.Black, PfColors.Green400, PfColors.Blue, PfColors.Orange400] }}
-              legend={{ show: true }}
-              grid={{ y: { show: false } }}
-              axis={axis}
-              data={chartData}
-            />
+            {(this.thereIsTrafficData() && (
+              <AreaChart
+                size={{ height: 80 }}
+                color={{ pattern: [PfColors.Black, PfColors.Green400, PfColors.Blue, PfColors.Orange400] }}
+                legend={{ show: true }}
+                grid={{ y: { show: false } }}
+                axis={axis}
+                data={chartData}
+              />
+            )) || (
+              <div>
+                <Icon type="pf" name="info" /> Not enough data.
```

Comment:
I understand why we want to change the text although I'm not sure I quite like 'Not enough data''.  It leaves me wondering, "not enough data for what exactly?".  I think maybe this is more descriptive, but get at least one more opinion:

"Not enough traffic to generate chart."

Reply:
I asked and was being picky with the wording. I didn't win the short discussion and your suggestion is good enough. So... I'm using your text.

Reply:
I don't really know what is good, not really a contest.  We can always update the text again later.